common/primitives/src/msa.rs

```diff
@@ -6,6 +6,9 @@ use serde::{Deserialize, Serialize};
 use sp_runtime::DispatchError;
 use sp_std::prelude::Vec;
 
+/// The gap between the current block and a future expiring block allowed when validating signature proofs.
+pub const EXPIRATION_BLOCK_VALIDITY_GAP: u32 = 128;
+
 /// Message Source Id or msaId is the unique identifier for Message Source Accounts
```

Comment:
Question: Will this number be required to be updated via runtime upgrades or via governance, then we should set this number in runtime instead of hardcoding it here
Check for example MaxKeys
https://github.com/LibertyDSNP/frequency/blob/27aa8738785a593a7ee6bd65a600e49783e27487/runtime/frequency/src/lib.rs#L346

Reply:
Hard code is fine I think.

Reply:
I could see this being more useful to be a modifiable value? Perhaps for now we can set it to 150 blocks as per @aramikm 's comment so that at 6 seconds per block, the gap is 15 minutes